### PR TITLE
Update README.md with up-to-date screenshots

### PR DESCRIPTION
**Pull request by Feichtmeier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,15 @@ More information is available at https://community.ubuntu.com/t/faq-ubuntu-new-t
 
 Detailed screenshots and reasoning behind the design can be found inside the Design Wiki: https://github.com/ubuntu/yaru/wiki
 
-![Files](https://i.imgur.com/XA00uS9.jpg)
+![Files](https://i.imgur.com/Qxl69M6.png)
```
Comment (clobrano) [inline README.md]: can we use the official wallpaper instead?

Comment (Feichtmeier) [reply README.md]: That is actually one of the wallpapers installed by default :) have a look

Comment (clobrano) [reply README.md]: Yes,  but I meant the one you have soon after install :)

Comment (Feichtmeier) [reply README.md]: done

**Pull request merged**